core/src/modules/plugins.ts

```diff
@@ -42,6 +42,17 @@ export namespace Plugins {
     };
   }
 
+  export type GrouparooManifestPackage = {
+    name: string;
+    description: string;
+    imageUrl: string;
+    packageName: string;
+    source: boolean;
+    destination: boolean;
+    npmUrl?: string;
+    docsUrl?: string;
+  };
+
```

Comment:
Finally typing the plugin data we get from https://www.grouparoo.com/plugins/v1/manifest.json